Source/Core/InputCommon/ControllerInterface/DInput/DInputJoystick.cpp

```diff
@@ -46,28 +46,25 @@ void InitJoystick(IDirectInput8* const idi8, HWND hwnd)
                   js_device->SetCooperativeLevel(nullptr, DISCL_BACKGROUND | DISCL_NONEXCLUSIVE)))
           {
             // PanicAlert("SetCooperativeLevel failed!");
-            js_device->Release();
             continue;
           }
         }
 
-        auto js = std::make_shared<Joystick>(js_device);
+        auto js = std::make_shared<Joystick>(js_device.Get());
```

Comment:
Like I said in the other PR, this should probably be passed all the way in if the device is just going to take the raw pointer and then create a shared pointer from it